src/components/Popups/ListUpdatePopup.tsx

```diff
@@ -86,10 +86,10 @@ export default function ListUpdatePopup({
             </div>
             <AutoRow>
               <div style={{ flexGrow: 1, marginRight: 12 }}>
-                <ButtonSecondary onClick={handleAcceptUpdate}>Accept update</ButtonSecondary>
```

Comment:
ButtonSecondary had a background of `primaryDark`. We don't have that style so we'll have to come back to it if the normal Primary doesn't look good.